### PR TITLE
feat(tier4_system_msgs): add mrm behavior status msg and operation srv

### PR DESCRIPTION
**Pull request by mkuri**
Signed-off-by: Makoto Kurihara <mkuri8m@gmail.com>

## Related Links

https://github.com/autowarefoundation/autoware.universe/pull/2070
https://github.com/autowarefoundation/autoware.universe/issues/1856

## Description

Add a message for each MRM operator informing their execution and availability status

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code is properly formatted
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code is properly formatted
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: tier4_system_msgs/CMakeLists.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "msg/HazardStatusStamped.msg"
   "msg/ServiceLog.msg"
   "msg/IsAutonomousAvailable.msg"
+  "msg/MRMBehaviorStatus.msg"
```
Comment (kenji-miyake) [inline tier4_system_msgs/CMakeLists.txt]: Same as https://github.com/autowarefoundation/autoware_adapi_msgs/pull/16#discussion_r1000179751.

```suggestion
  "msg/MrmBehaviorStatus.msg"
```

File: tier4_system_msgs/msg/MRMBehaviorStatus.msg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+builtin_interfaces/Time stamp
+bool is_available
+bool is_operating
```
Comment (kenji-miyake) [inline tier4_system_msgs/msg/MRMBehaviorStatus.msg]: Do you mean you want to get these statuses independently?

Could you show me some use cases that the following definition can't support?

```
NOT_AVAILABLE=0
AVAILABLE=1
OPERATING=2

int8 state
```

Comment (mkuri) [reply tier4_system_msgs/msg/MRMBehaviorStatus.msg]: There is no situation that is (!is_available && is_operating), so change as above.

Comment (mkuri) [reply tier4_system_msgs/msg/MRMBehaviorStatus.msg]: Fixed the state in the following commit.
https://github.com/tier4/tier4_autoware_msgs/pull/60/commits/64e7f6008983144ccb5a3905a4dc44ae48cf425d

File: tier4_system_msgs/srv/OperateMrm.srv [hunk 3]
```diff
@@ -0,0 +1,4 @@
+builtin_interfaces/Time stamp
+bool operate
+---
+autoware_adapi_v1_msgs/ResponseStatus response
```
Comment (isamu-takagi) [inline tier4_system_msgs/srv/OperateMrm.srv]: Please use [autoware_common_msgs/ResponseStatus](https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_common_msgs/msg/ResponseStatus.msg) for internal interface services.

Comment (mkuri) [reply tier4_system_msgs/srv/OperateMrm.srv]: Fixed to use autoware_common_msgs in the following commit.
https://github.com/tier4/tier4_autoware_msgs/pull/60/commits/97e48f8c8c0c6b65423e871492772400656b1ca4

**Review by isamu-takagi — APPROVED** [review]
LGTM

**Pull request merged**